backend/src/main/java/com/empire/Region.java

```diff
@@ -272,20 +330,24 @@ public Node(double power, Region location) {
 		for (final Character c : w.characters) {
 			Function<Node, Node> getPower = (Node n) -> {
 				Region r = n.location;
-				if (r.isSea()) return new Node(n.power * .9, n.location);
+				if (r.isSea()) return new Node(n.power * Constants.plotDecaySea, n.location);
 				if (NationData.isFriendly(c.kingdom, r.kingdom, w)) {
 					if (r.religion == Ideology.LYSKR) return new Node(n.power, n.location);
-					return new Node(n.power * (.9 - r.calcUnrest(w) / 10), n.location);
+					return new Node(n.power * (Constants.plotDecayFriendly - r.calcUnrest(w) / 10), n.location);
 				}
-				return new Node(n.power * (.8 + r.calcUnrest(w) / 10), n.location);
+				return new Node(n.power * (Constants.plotDecayNonFriendly + r.calcUnrest(w) / 10), n.location);
 			};
+
+			// TODO: try this function-style equivalent instead once unit tests are created
+			//PriorityQueue<Node> pq = new PriorityQueue<>(100, Comparator.comparingDouble(n -> n.power));
```

Comment:
This sorts descending, so you'd actually want -n.power, I think, or use (a,b)  -> Double.compare(b, a) as the Comparator.

Reply:
I added the negative for now but didn't make the change, will do it when writing the unit tests for this guy.